src/logging.h

```diff
@@ -8,8 +8,11 @@
 
 #include <fs.h>
 #include <tinyformat.h>
+#include <util/macros.h>
+#include <util/time.h>
```

Comment:
Might want to put the timer in a new header. Pulling in these extra header files in everything that uses `logging.h` (read: everything) might slow down the build unnecessarily.

Reply:
Last time I looked, I found that the standard library headers are generally fast to include (compared to boost and our serialize.h)

Reply:
In comparison that's true. Still it's good to be extra careful about introducing extra dependencies for such a common header.